go.mod

```diff
@@ -8,14 +8,21 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/jpillora/backoff v1.0.0 // indirect
 	github.com/mitchellh/go-homedir v1.1.0 // indirect
-	github.com/onsi/gomega v1.25.0
+	github.com/onsi/gomega v1.27.4
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.0 // indirect
 	github.com/stretchr/testify v1.8.2
-	k8s.io/api v0.26.2
-	k8s.io/apimachinery v0.26.2
-	k8s.io/client-go v0.26.2
-	sigs.k8s.io/controller-runtime v0.14.4
+	k8s.io/api v0.27.1
+	k8s.io/apimachinery v0.27.1
+	k8s.io/client-go v0.27.1
+	sigs.k8s.io/controller-runtime v0.14.6
+)
+
+// TODO: remove once client-go v0.27.2 is released: https://github.com/kubernetes-sigs/controller-runtime/issues/2302
```

Comment:
what is that for ?

Reply:
which dependency has triggered this update?

Reply:
We need to upgrade to the new version. But at the moment, it's incompatible with controller runtime. Once they release 0.27.2, we can remove the replace.